Automattic/wp-calypso
# Offer Reset: configure product cards when products are owned or included as a plan feature

#### Changes proposed in this Pull Request

* Make product cards looks as per design when products are owned or included as a plan feature. Specifically, the badge label should be either "Included in your plan" or "You own this", and the CTA should move users to the purchases page (in both cases).

#### Testing instructions

* Run this PR with Offer Reset AB experiment enabled.
* Go to the Plans page.
* Purchase Jetpack Security Daily.
* Go to the Plans page.
* Verify that Jetpack Backup Daily, Jetpack Scan Daily, and Jetpack Anti-spam products are tagged with the "Included in your plan" badge, and that CTAs point to the me/purchases page.
* Repeat this process with other plans and make sure that product cards reflect the real relationship their have with the site purchases.

Fixes 1169247016322522-as-1189986731974009

#### Demo
##### Owning Jetpack Security Daily
![Security Daily](https://user-images.githubusercontent.com/3418513/90932140-6c6ee400-e3d4-11ea-8326-d5843faf7a0a.gif)

##### Owning Jetpack Security Real-time
![Security RT](https://user-images.githubusercontent.com/3418513/90932026-36c9fb00-e3d4-11ea-87b5-0bfcda94afba.gif)

##### Owning Jetpack Backup Daily
![Backup Daily](https://user-images.githubusercontent.com/3418513/90932463-18b0ca80-e3d5-11ea-86ee-c7046940f8d0.gif)
